src/lib/modules/Literature/LiteratureRelations.js

```diff
@@ -24,50 +24,53 @@ export default class LiteratureRelations extends Component {
   renderMultiparts = () => {
     const relations = _get(this.relations, 'multipart_monograph', []);
     if (!relations.length) return null;
+    const prefix = 'This is part of the monograph:';
 
-    const cmp = relations.map((rel) => {
-      const volume = _get(rel, 'volume');
-      const text = volume
-        ? `${rel.record_metadata.title} (vol: ${volume})`
-        : rel.record_metadata.title;
-      return (
-        <React.Fragment key={rel.pid_value}>
-          This is part of the monograph{' '}
-          <Link to={this.getLinkTo(rel)}>
-            <LiteratureTitle title={text} />
-          </Link>
-        </React.Fragment>
-      );
-    });
-    return <List.Item>{cmp}</List.Item>;
+    return (
+      <List.Item>
+        {prefix}
+        <List bulleted>
+          {relations.map((rel) => {
+            const volume = _get(rel, 'volume');
+            const text = volume
+              ? `${rel.record_metadata.title} (vol: ${volume})`
+              : rel.record_metadata.title;
+            return (
+              <List.Item bulleted key={rel.pid_value}>
+                <Link to={this.getLinkTo(rel)}>
+                  <LiteratureTitle title={text} />
+                </Link>
+              </List.Item>
+            );
+          })}
+        </List>
+      </List.Item>
+    );
   };
 
   renderSerials = () => {
     const relations = _get(this.relations, 'serial', []);
     if (!relations.length) return null;
-
-    const items = relations.map((rel) => {
-      const volume = _get(rel, 'volume');
-      const text = volume
-        ? `${rel.record_metadata.title} (vol: ${volume})`
-        : rel.record_metadata.title;
-      return (
-        <React.Fragment key={rel.pid_value}>
-          <Link to={this.getLinkTo(rel)}>
-            <LiteratureTitle title={text} />
-          </Link>
-        </React.Fragment>
-      );
-    });
+    const prefix = 'This is part of the periodical: ';
 
     return (
       <List.Item>
-        <SeparatedList
```

Comment:
is SeparatedList component used anywhere else? Wouldn't it be broken visually also in the other places?

Reply:
yes it used still even in the same component, i think for longer texts it is easy to break but otherwise it works fine.